packages/formation-react/src/components/ErrorableCheckboxGroup/ErrorableCheckboxGroup.stories.jsx

```diff
@@ -11,7 +11,11 @@ export default {
   },
 };
 
-const Template = (args) => <ErrorableCheckboxGroup {...args} />;
+const Template = (args) => (
+  <div style={{ paddingLeft: '1em' }}>
+    <ErrorableCheckboxGroup {...args} />
+  </div>
+);
```

Comment:
Added some padding to the left so we can see the red bar for the error.

Reply:
I usually opt for `rem` units, but I don't know if we have a standardized unit (and maybe we should).